next.config.js

```diff
@@ -87,4 +88,20 @@ module.exports = {
       },
     ];
   },
+  async redirects() {
+    return [
+      {
+        source: "/",
+        has: [
+          {
+            type: "header",
+            key: "User-Agent",
+            value: "(.*Trident.*)",
+          },
+        ],
+        destination: "/notsupported",
+        permanent: false,
+      },
+    ];
+  },
```

Comment:
This will only redirect you if you navigate to the splash page, it would be nice if the redirect happened from any page.

Reply:
Given the fact that a catch-all regex pattern seems to prevent pages from rendering on IE (though it does redirect properly), I've added a rule for every page that checks if IE is being used. This is obviously not the most ideal since we'd need to add a new rule every time we add a new page, but it is the best we can do for the time being.

I also added IE detection to the 404 and 500 pages since they were not auto redirecting using the redirects function